docs/setup/kubeedge_cri_configure.md

```diff
@@ -65,7 +65,7 @@ CONTAINER                                                           IMAGE
 41c1a07fe7bf7425094a9b3be285c312127961c158f30fc308fd6a3b7376eab2    docker.io/library/nginx:1.15.12    io.containerd.runtime.v1.linux
 ```
 
-*NOTE: namespace here might be changed soon from “k8s.io” to “kubeedge.io”.*
+NOTE: since cri doesn't supports multi-tenancy while containerd does, namespace for containers are set to "k8s.io" defaultly and no way to change that until the [cri's support](https://github.com/containerd/cri/pull/1462) has done.
```

Comment:
```suggestion
NOTE: since cri doesn't support multi-tenancy while containerd does, namespace for containers are set to "k8s.io" defaultly and no way to change that until the [cri's support](https://github.com/containerd/cri/pull/1462) has done.
```